test/test_dynamic_shapes.py

```diff
@@ -83,6 +83,23 @@ def test_sizeAdd(self):
     t4 = t3.expand(dyn_size)
     self.assertEqual(t4.size(0), 3)
 
+  def test_sizeGe(self):
+    met.clear_all()
+
+    size1 = 5
+    size2 = 2
+    t1 = torch.zeros([size1, size2], device=dev)
+    t1[3][0] = 1
+    # t2 has size [<=10, 2]
+    t2 = torch.nonzero(t1)
+    # Create a SizeAdd IR node.
+    # t2.shape[1] generates a SizeConstant node.
+    dyn_size = t2.shape[0] >= t2.shape[1]
+    self.assertGreater(met.counter_value("xla::size_ge"), 0)
```

Comment:
@vanbasten23 @miladm @JackCaoG I enhanced the dynamic ir with counters and therefore we can test that like the cpp tests here. Does that sound okay to you?